docs-src/ref-programs.scrbl

```diff
@@ -2297,3 +2297,33 @@ it does not matter who @reachin{publish}es, such as in a @reachin{timeout}.
 @reachin{Anybody} is strictly an abbreviation of a @reachin{race} involving all of the named participants of the application.
 In an application with a @tech{participant class}, this means any principal at all, because there is no restriction on which principals (i.e. addresses) may serve as a member of that class.
 In an application without any @tech{participant class}es, @reachin{Anybody} instead would mean only the actual previously-bound @tech{participant}s.
+
+@subsubsection{'use strict'}
+
+@(mint-define! '("'use strict'"))
+@reach{
+  'use strict'; }
+
+@index{'use strict'} @reachin{'use strict'} enables unused variables checks for all subsequent
+declarations within the current scope. If a variable is declared, but never used, there will
+be a warning emitted at compile time.
```

Comment:
Do you feel strongly about warning versus error? I assumed we'd do error.

Reply:
Personally, if my program will do what I want, despite a little bloat, I'd like it to compile 🙂 but I don't feel super strong about it, I can make it error if you want

Reply:
I think I'd like error